src/pytest/test_followhashes.py

```diff
@@ -463,5 +463,26 @@ def testExpire(env):
     conn.execute_command('HSET', 'doc1', 'test', 'foo')
     env.expect('FT.SEARCH idx foo').equal([1L, 'doc1', ['test', 'foo']])
     conn.execute_command('EXPIRE', 'doc1', '1')
-    sleep(1.1)
+    sleep(0.8)
```

Comment:
You do not need it, its enough to check right after without the sleep (and I think it will be safer with respect to timings)